### PR TITLE
always call the callback and return

### PR DESCRIPTION
**Pull request by pibi**
Fix for #12

### CODE REVIEW

**Comment by Amri91** [comment]
@pibi, I tested the new code and I am still getting the issue mccormicka/Mockgoose#204.

Do we not need to use something like this?

```
if(typeof(callback) == "function"){
            callback(child.status);
}
```

at the end?

File: index.js [hunk 1]
```diff
@@ -77,24 +77,20 @@ function start_server(opts, callback) {
...
+            // spawn a mongo killer
             debug('starting mongokiller.js, ppid:%d\tmongod pid:%d', process.pid, child_pid);
             var killer = proc.spawn("node", [path.join(__dirname, "binjs", "mongokiller.js"), process.pid, child_pid], {
                 stdio: 'ignore',
                 detached: true
             });
             killer.unref();
         }
-
+        callback(child.status);
```
Comment (Amri91) [inline index.js:93]: We still need an IF statement to check if the callback is indeed a function.

**Comment by brandon-arnold** [comment]
I also required the conditional described by @Amri91. I'm on Windows 10.

My build of MERN is broken without this. When will it get merged?

**Comment by winfinit** [comment]
there was a rewrite of this library, so I am closing this pull request.

**Pull request closed without merging**